nixos/modules/services/databases/cockroachdb.nix

```diff
@@ -9,8 +9,9 @@ let
   escape    = builtins.replaceStrings ["%"] ["%%"];
   ifNotNull = v: s: optionalString (v != null) s;
 
-  startupCommand = lib.concatStringsSep " "
-    [ # Basic startup
+  startupCommand = lib.escapeShellArgs " "
```

Comment:
```suggestion
  startupCommand = lib.escapeShellArgs
```

Reply:
Good catch, I amended the fix to my commit :)